spec/models/top_movie_set_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe TopMovieSet, type: :model do
+  describe 'instance_methods' do
+    it 'count' do
```

Comment:
If two models have this method and it is the same method, maybe put it in the application_record? Not completely convinced but feels like a possibility

Reply:
I like it.  However because these are poros I think method would need to fall into a mixin module.  Should we hold off until George has done the refactor and then add then?

Reply:
Yes, I think that's probably wise...

Reply:
Yep!